src/index.js

```diff
@@ -125,6 +130,18 @@ class Dropzone extends React.Component {
     }
   }
 
+  handleOnCancel() {
+    const { close } = this.props;
+    const { fileInputEl } = this;
+    setTimeout(() => {
+      const FileList = fileInputEl.files;
+      if (!FileList.length) {
+        document.body.onfocus = null;
```

Comment:
Same as with `addEventListener`, you should use `removeEventListener` here. What I don't get is, why are you removing it every time but adding just once?